wcfsetup/install/files/lib/system/cronjob/CronjobScheduler.class.php

```diff
@@ -148,7 +148,9 @@ protected function resetFailedCronjobs() {
 							'execTime' => TIME_NOW
 						]);
 						$logEditor = new CronjobLogEditor($log);
-						$this->logResult($logEditor, new \Exception("Cronjob stuck in state '".$cronjob->state."' for two periods (nextExec '".$cronjob->nextExec."', afterNextExec '".$cronjob->afterNextExec."', now '".TIME_NOW."')."));
+						
+						$errorMessage = sprintf("The cronjob '%s' (ID %d) appears to have failed. (nextExec %d, afterNextExec %d, now %d)", $cronjob->cronjobID, $cronjob->cronjobName, $cronjob->nextExec, $cronjob->afterNextExec, TIME_NOW);
```

Comment:
```suggestion
						$errorMessage = sprintf("The cronjob '%s' (ID %d) appears to have failed. (nextExec %d, afterNextExec %d, now %d)", $cronjob->cronjobName, $cronjob->cronjobID, $cronjob->nextExec, $cronjob->afterNextExec, TIME_NOW);
```
The first 2 parameters were in the wrong order.